### PR TITLE
408 ci update publish workflow w leapfrogai UI artifacts

### PR DESCRIPTION
**Pull request by jalling97**
- Adds the UI zarf build process to the release workflow
- Updates the Makefile with new ui target
- Updates UI zarf package yamls

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,6 +53,18 @@ build-api: local-registry setup-api-deps ## Build the leapfrogai_api container a
 	uds zarf package create packages/api -o packages/api --registry-override=ghcr.io=localhost:5000 --insecure --set LEAPFROGAI_IMAGE_VERSION=${LOCAL_VERSION} --confirm
 
 
+build-ui: ## Build the leapfrogai_ui container and Zarf package
```
Comment (YrrepNoj) [inline Makefile:56]: With this new make target, I tthink  it make sense to remove the Makefile at `src/leapfrogai_ui/Makefile`.

**Review by YrrepNoj — CHANGES_REQUESTED** [review]
Thanks for the PR. Just one single change request!

File: Makefile (hunk 1 above)
Comment (jalling97) [reply Makefile:56]: Good catch. I'll remove the old one and update any documentation referencing it

Comment (jalling97) [reply Makefile:56]: Removed :+1:

**Review by YrrepNoj — APPROVED** [review]

**Pull request merged**